### PR TITLE
Added error loggs handling

### PR DESCRIPTION
**Pull request by esuwu**

### CODE REVIEW

File: cmd/forkdetector/internal/api.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func (a *api) Start() <-chan struct{} {
 		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 		err := a.srv.Shutdown(ctx)
 		if err != nil {
-			zap.S().Errorf("Failed to shutdown API server: %v", err)
+			zap.S().Infof("Failed to shutdown API server: %v", err)
```
Comment (alexeykiselev) [inline cmd/forkdetector/internal/api.go:110]: No need to change it here.

File: cmd/wmd/internal/api.go [hunk 2]
```diff
@@ -99,7 +99,7 @@ func NewDataFeedAPI(interrupt <-chan struct{}, logger *zap.Logger, storage *stat
 		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 		err := apiServer.Shutdown(ctx)
 		if err != nil {
-			zap.S().Errorf("Failed to shutdown API server: %v", err)
+			zap.S().Infof("Failed to shutdown API server: %v", err)
```
Comment (alexeykiselev) [inline cmd/wmd/internal/api.go:102]: No need to change it here

File: pkg/api/node_api.go [hunk 3]
```diff
@@ -196,7 +196,7 @@ func Run(ctx context.Context, address string, n *NodeApi) error {
 		zap.S().Info("Shutting down API...")
 		err := apiServer.Shutdown(ctx)
 		if err != nil {
-			zap.S().Errorf("Failed to shutdown API server: %v", err)
+			zap.S().Infof("Failed to shutdown API server: %v", err)
```
Comment (alexeykiselev) [inline pkg/api/node_api.go:199]: Better to put it on WARN level

File: pkg/node/node.go [hunk 4]
```diff
@@ -100,6 +100,15 @@ func (a *Node) Serve(ctx context.Context) error {
 	}
 }
 
+func (a *Node) LogErrors(err error) {
```
Comment (alexeykiselev) [inline pkg/node/node.go:103]: Definitely this is not a public function. May be there is no need for a separate function at all if it used once...

**Review by alexeykiselev — CHANGES_REQUESTED** [review]

File: pkg/node/node.go (hunk 4 above)
Comment (esuwu) [reply pkg/node/node.go:103]: You're right but the function "Run" is big enough I'll make that method private

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**